.github/ISSUE_TEMPLATE/issue--bug-report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: 'Issue: Bug report'
+about: Create a report to help us improve Hazel
+title: Add a relevant bug report title here
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+A screenshot or copy of the error could be helpfull as well.
```

Comment:
Helpfull -> Helpful

Reply:
Fixed typo in https://github.com/TheCherno/Hazel/pull/80/commits/bda4f8f9e22f8887fb4b046c185349f9863368a4.